compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/Expect.kt

```diff
@@ -50,6 +49,6 @@ internal expect fun removePost(token: Any?)
  */
 internal expect abstract class PlatformOptimizedCancellationException(
     message: String? = null
-) : CancellationException
+) : kotlinx.coroutines.CancellationException
```

Comment:
Why do we remove import in here? Let's try to avoid changes in commonMain in fork, especially in cases where it is not necessary.

Reply:
didn't compile. I'll create a YT ticket

Reply:
Do we have issue with this particular import? I can understand that it was an issue with [kotlin.coroutines.cancellation.CancellationException -> kotlinx.coroutines.CancellationException](https://github.com/JetBrains/compose-multiplatform-core/pull/1776/files#diff-ec59e865eaf971ac8f3379b4a976728a23dc9a027356c5f5b553afb72777596cL22), but can be this line changed by mistake?

Reply:
Right. I fixed that.

Reply:
We need to fix it `integration` as well. otherwise the next merge it will be restored

Reply:
Yes, I thought that integration will be merged into jb-main eventually. But that won't happen as we discussed with Ivan.